src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-connector.html

```diff
@@ -0,0 +1,31 @@
+<div>
+
+    <p>Basically, for all connection methods, Jenkins will start by triggering a <code>docker run</code>.
+        Then, after this step, there will optionally be more steps to establish the connection.
+        There is currently three alternative ways to connect your Jenkins master
+        to the dynamically provisioned Docker agents.
+    </p>
+
+    <p>There are different pros and cons for each connection method. Depending on your environment, choose the one matching your needs. More detailed
+        prerequisites are provided once you select a given method.</p>
+
+    <dl>
+        <dt>Attach Docker container</dt>
+        <dd>This method runs a container, then connects to it using <code>docker exec</code>,
+            all by using the Docker API.
+            The agent does not need to be able to reach the master through the network layers
+            to communicate, all will go through Docker API.
```

Comment:
Nit: Should be semicolon, not comma; we're making a separate, but related, statement.
(or one could use a - instead)